frontend/packages/core/src/Input/select.tsx

```diff
@@ -300,4 +300,128 @@ export const Select = ({
   );
 };
 
-export default Select;
+export interface MultiSelectProps extends Pick<MuiSelectProps, "disabled" | "error"> {
+  defaultOptions?: Array<number> | Array<string>;
+  helperText?: string;
+  label?: string;
+  name: string;
+  selectOptions: SelectOption[];
+  onChange?: (values: Array<string>) => void;
+  flex?: boolean;
+}
+
+const MultiSelect = ({
+  defaultOptions = [],
+  disabled,
+  error,
+  helperText,
+  label,
+  name,
+  selectOptions,
+  onChange,
+  flex = false,
+}: MultiSelectProps) => {
+  // Flattens all options and sub grouped options for easier retrieval
+  const flatOptions: BaseSelectOptions[] = flatten(
```

Comment:
We should probably pull this out into a helper function since its now duplicated